smartsim/experiment.py

```diff
@@ -467,35 +467,35 @@ def summary(self):
         The summary will show each instance that has been
         launched and completed in this ``Experiment``
 
-        :return: pandas Dataframe of ``Experiment`` history
-        :rtype: pd.DataFrame
+        :return: tabulate string of ``Experiment`` history
```

Comment:
Lets allow ``format`` as a parameter here and pass to the tabulate function. Also should add in https://github.com/astanin/python-tabulate#table-format to the docstring so that people know what options are available.